tooling/What-We-Use.md

```diff
@@ -0,0 +1,20 @@
+#What We Use
+
+##Commonly used tools on the team in alphabetical order
+
+(TODO: link each item to either a separate documentation file or an external page)
+
+| Name | Description | Category |
+| ---- | ----------- | ---|
+|Cisco Connect|Broad VPN|IT|
+|Github/Github desktop|Version control|Code|
+|Google calendar|Scheduling/Shared calendars|Team|
+|IntelliJ|Code editor|Code|
+|Jira|Sprint kanban board|Scrum|
```

Comment:
The Kanban board is not currently in use so I would recommend removing 'kanban' from the description here

Reply:
Is Jira itself not a Kanban board? I'd always heard "Kanban" as a general term for that kind of ticketing system/display board, but maybe that's not right. I agree that having slightly more detail in the description would be good.

Reply:
Kanban and Scum are two styles of Agile, and Jira supports boards for both of these. We had a Kanban for awhile and I think it's still linked in our Jira Project which is why it might be confusing to have Kanban in the description for Jira, as we should only be using the Sprint/Scrum board at this point.

Reply:
Gotcha, I didn't realize it had a more specific usage! I'll update that.